docker/buildDocker.sh

```diff
@@ -68,9 +68,9 @@ checkJDKVersion()
 		"jdk13u" | "jdk13" | "13" | "13u" )
 			jdkVersion="jdk13u";;
                 "jdk14u" | "jdk14" | "14" | "14u" )
-                        jdkVersion="jdk14u";;
+                        jdkVersion="jdk14";;
                 "jdk15u" | "jdk15" | "15" | "15u" )
-                        jdkVersion="jdk15u";;
+                        jdkVersion="jdk15";;
```

Comment:
Are you sure this is correct? Given the jdk14u problem and the fact we don't have an `openjdk-jdk15` repo I suspect this might have to be `jdk`

Reply:
That's a good point - I put that in more as a future-proof, as I didn't think it'd be used in DF Check job for a little bit. Forgot that `jdk` could be put in to build the next one.

Reply:
Commit in.